pkgs/development/interpreters/lua-5/interpreter.nix

```diff
@@ -132,6 +133,9 @@ stdenv.mkDerivation rec {
     ln -s "$out/lib/pkgconfig/lua.pc" "$out/lib/pkgconfig/lua-${luaversion}.pc"
     ln -s "$out/lib/pkgconfig/lua.pc" "$out/lib/pkgconfig/lua${luaversion}.pc"
     ln -s "$out/lib/pkgconfig/lua.pc" "$out/lib/pkgconfig/lua${lib.replaceStrings [ "." ] [ "" ] luaversion}.pc"
+
+    # Make documentation outputs of different versions co-installable.
+    mv $out/share/doc/lua $out/share/doc/lua-$version
```

Comment:
```suggestion
    mv $out/share/doc/lua $out/share/doc/lua-${version}
```

Reply:
We should use nix variables if possible and that problem could be solved with finalAttrs.